From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lulu13022002 <41980282+Lulu13022002@users.noreply.github.com>
Date: Sun, 3 Mar 2024 18:05:10 +0100
Subject: [PATCH] server gen test


diff --git a/build.gradle.kts b/build.gradle.kts
index 2e9820330df75757007cfc70ad35bdcad957adaf..6535f20cb17766885090eee4025f77fe75ca2d6d 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -175,14 +175,18 @@ fun TaskContainer.registerRunTask(
 }
 
 // Paper start
+val generatedServerFolder: File = rootProject.file("paper-api-generator/generatedServerTest")
+
 sourceSets.main {
     java {
+        srcDir(generatedServerFolder)
         exclude("excluded/**")
     }
 }
 
 idea {
     module {
+        generatedSourceDirs.add(generatedServerFolder)
         excludeDirs.add(projectDir.resolve("src/main/java/excluded"))
     }
 }
